Assets/Scripts/UI/DraggableObject.cs

```diff
@@ -184,9 +191,10 @@ public void SetSnapToAreas(List<SnapLocation> snapLocations)
         snapToAreas = snapLocations;
     }
 
-    public void SetIsDraggable(bool draggable)
+    public void SetIsDraggable(bool draggable, Vector3 scale)
```

Comment:
Let's have only one argument for DraggableObject.SetIsDraggable. We may want to modify draggability separately from the scale. Adjusting the scale values should have their own methods rather than being a part of this one.